src/main/java/org/example/postory/domain/post/repository/PostRepository.java

```diff
@@ -1,5 +1,27 @@
 package org.example.postory.domain.post.repository;
 
-public class PostRepository {
+import java.util.List;
+import java.util.stream.Collectors;
+import org.example.postory.domain.post.dto.PostResponseDto;
+import org.example.postory.domain.post.entity.Post;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
+public interface PostRepository extends JpaRepository<Post, Long> {
+
+    //공개 게시글 + 삭제되지 않은 게시글 + 수정일 기준 최신순 정렬
+    @Query(value = "SELECT Post FROM Post "
+        + "WHERE user = :user_id "
+        + "AND isPublic = TRUE "
+        + "AND deletedAt IS NULL "
+        + "ORDER BY updatedAt DESC ")
```

Comment:
```suggestion
    @Query(value = """SELECT Post FROM Post
        WHERE user = :user_id
        AND isPublic = TRUE
        AND deletedAt IS NULL
        ORDER BY updatedAt DESC 
        """)
```
이런 식으로 자바 텍스트 블럭 사용해보시면 좋을 것 같습니다!